freyja/utils.py

```diff
@@ -1,13 +1,17 @@
-import matplotlib.pyplot as plt
-import pandas as pd
-import re
 import copy
-from datetime import datetime
+import os
+import re
+import tqdm
+
 import matplotlib.dates as mdates
-import plotly.graph_objects as go
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
 import plotly.express as px
-import os
+import plotly.graph_objects as go
+from datetime import datetime
 import yaml
+from scipy.optimize import curve_fit
```

Comment:
if we haven't added scipy to the list of required packages, it should be added: https://github.com/andersen-lab/Freyja/blob/main/ci/conda_requirements.txt